### PR TITLE
31082 social links styling

### PR DESCRIPTION
**Pull request by bienvu**
Add a social links component with a link for each social network site.

https://redmine.previousnext.com.au/issues/31082

### CODE REVIEW

File: themes/agov/agov_whitlam/sass/_variables.scss [hunk 1]
```diff
@@ -246,6 +246,10 @@ $chroma: add-colors('functional', (
   form-element-bg-hover:      'white-smoke',
 
   breadcrumb-color:       'grey-medium',
+
+  social-bg:              'white-smoke',
+  social-bg-hover:        'grey-extra-light',
+  social-icon:            'steelblue',
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/_variables.scss]: There's no such color as steelblue. This is causing the Sass to fail to build.

**Comment by bienvu** [comment]
Ok.thanks.
I removed var color steelblue and used blue-steel.

**Comment by kimpepper** [comment]
test this please

**Comment by kimpepper** [comment]
Feedback has been resolved.

**Pull request merged**